webapp/src/domain/Place.tsx

```diff
@@ -0,0 +1,59 @@
+export default class MapPoint {
```

Comment:
Please rename the file Place.tsx to MapPoint.tsx in order to be transparent to the class inside it.